src/System.Security.Cryptography.X509Certificates/tests/CertificateCreation/CertificateRequestChainTests.cs

```diff
@@ -201,6 +218,85 @@ public static void ChainCertRequirements(bool useIntermed, bool? isCA, X509KeyUs
             }
         }
 
+        [Fact]
+        public static void BuildChainUntrustedRoot()
+        {
+            using (RSA rootPrivKey = RSA.Create(3072))
+            using (RSA intermed1PrivKey = RSA.Create(2048))
+            using (RSA leafKey = RSA.Create(1536))
+            using (RSA leafPubKey = RSA.Create(leafKey.ExportParameters(false)))
+            {
+                leafPubKey.ImportParameters(leafKey.ExportParameters(false));
+                CreateCertificates(rootPrivKey, intermed1PrivKey, intermed1PrivKey, leafPubKey, BuildAndTestChainLocal, false);
+            }
+
+            void BuildAndTestChainLocal(
+                X509Certificate2 rootCert,
+                X509Certificate2 intermed1Cert,
+                X509Certificate2 intermed2Cert,
+                X509Certificate2 leafCert,
+                bool rootExtraStore)
+            {
+                using (X509Chain chain = new X509Chain())
+                {
+                    chain.ChainPolicy.ExtraStore.Add(rootCert);
+                    chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
+                    chain.ChainPolicy.VerificationTime = DateTimeOffset.UtcNow.ToLocalTime().DateTime;
```

Comment:
The VerificationTime value is already the Now value from when the Chain(Policy) was built.  The purpose of setting it explicitly in previous tests was to avoid clock skew, so here it should be set as something like `leafCert.NotBefore.AddSeconds(1)`